pkgs/by-name/or/orthanc/package.nix

```diff
@@ -0,0 +1,113 @@
+{
+  lib,
+  stdenv,
+  fetchhg,
+  boost,
+  charls,
+  civetweb,
+  cmake,
+  curl,
+  dcmtk,
+  gtest,
+  jsoncpp,
+  libjpeg,
+  libpng,
+  libuuid,
+  log4cplus,
+  lua,
+  openssl,
+  protobuf,
+  pugixml,
+  python3,
+  sqlite,
+  unzip,
+  versionCheckHook,
+  nixosTests,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "orthanc";
+  version = "1.12.6";
+
+  src = fetchhg {
+    url = "https://orthanc.uclouvain.be/hg/orthanc/";
+    rev = "Orthanc-${finalAttrs.version}";
+    hash = "sha256-1ztA95PiCGL1oD6zVfsEhwrwGNID13/NcyZDD3eHYv0=";
+  };
+
+  patches = [
+    # Without this patch, the build fails to find `GOOGLE_PROTOBUF_VERIFY_VERSION`
+    ./add-missing-include.patch
+  ];
+
+  sourceRoot = "${finalAttrs.src.name}/OrthancServer";
+
+  nativeBuildInputs = [
```

Comment:
Those should be split into nativeBuildInputs and buildInputs

Reply:
How do you differentiate them? IMO these are dependencies for the build.

Reply:
Yes that's OK, but that list is libraries are required at build-time. I guess.

Reply:
Everything except unzip and cmake should buildInputs. Also try setting strictDeps = true to notice when things are used wrongly early.

Reply:
Thanks, I fixed it.